### ISSUE TITLE
Strange height of backquotes in SQL in PHP string literals

### ISSUE THREAD

**Issue by f1amy** [JetBrains/JetBrainsMono#411]
The backquotes have different height on start and end of names:
![image](https://user-images.githubusercontent.com/27062841/107502212-37d92980-6bba-11eb-800d-6b6a141e2984.png)

Looks like it depends on the string inside the quotes:
![image](https://user-images.githubusercontent.com/27062841/107502639-bafa7f80-6bba-11eb-8168-0160844a7127.png)

**Comment by philippnurullin** [comment]
Hi @f1amy . Please send me the exact text with symbols. Looks like the problem is with `grave` (U+0060). And the placement of it is shifted because of OpenType Feature "Case Sensitive Forms".
What IDE you using?

**Comment by f1amy** [comment]
Hello, @philippnurullin.
Here is the file: [backquotes.txt](https://github.com/JetBrains/JetBrainsMono/files/5958251/backquotes.txt).
IDE is PHPStorm 2021.1 EAP.

**Comment by f1amy** [comment]
I realized that this could be due to EAP IDE I'm using, but I don't have any other installed currently.

**Comment by philippnurullin** [comment]
Yes, it may. We are exploring the ability to support OpenType Features.
We will investigate this issue. Thanks for the feedback.

**Comment by toothbrush7777777** [comment]
Note that I see the same in the browser (Firefox / Microsoft Edge / Chrome) and Visual Studio Code (only tested on Windows).
This appears to happen when the preceding (Latin?) letter is capitalised.

```sql
SELECT `SQL`, `TestID`, `sql`, `testid`, `test_id` FROM `a`
```

![image](https://user-images.githubusercontent.com/8356617/150989622-0db4395f-408a-4156-ab45-ea3748e70934.png)

**Comment by philippnurullin** [comment]
I have spotted the source of the problem. It's autogenerated OpenType feature.
Turned off for now.
